frontend/packages/core/src/AppLayout/logo.tsx

```diff
@@ -10,13 +11,16 @@ const rotate = keyframes`
   }
 `;
 
-const StyledSvg = styled.svg`
-  height: 40px;
-  width: 40px;
-  &:hover {
-    animation: ${rotate} 5s linear;
-  }
-`;
+const StyledSvg = styled.svg({
+  height: "32px",
+  width: "32px",
+  marginLeft: "8px",
+  marginRight: "8px",
```

Comment:
Looking at the designs I think this should be
```suggestion
  height: "48px",
  width: "48px",
  padding: "8px",
```

Reply:
updated!